stdlib/tempfile.pyi

```diff
@@ -163,12 +163,12 @@ else:
             dir: Optional[_DirT[AnyStr]] = ...,
         ) -> IO[Any]: ...
 
-class _TemporaryFileWrapper(IO[str]):
-    file: IO[str]
+class _TemporaryFileWrapper(Generic[AnyStr], IO[AnyStr]):
```

Comment:
CI fails, because inheriting from `IO[AnyStr]` requires `__next__`. Lots of IO-like things don't actually inherit from `IO`, and we could do that here too:

```suggestion
class _TemporaryFileWrapper(Generic[AnyStr]):
```

Alternatively, you can add back `__next__` temporarily, so that someone else can change it in a separate PR.